src/3p-frame.js

```diff
@@ -27,6 +28,11 @@ import {urls} from './config';
 import {setStyle} from './style';
 import {domFingerprint} from './utils/dom-fingerprint';
 
+/**
+ * If true, then in experiment where the passing of context metadata
+ * has been moved from the iframe src hash to the iframe name attribute.
+ */
+const iframeContextInName = isExperimentOn(self, 'move-context-to-name');
```

Comment:
the string literal needs to be identical to experiment ID you defined in `experiments.js`.
So, `'move-context-to-name'` => `'3p-frame-context-in-name'`